Update Tenkuu Danzai Skelter Heaven (primary).ass

Shirotsumekusa opened the pull request:
Primary edit done

Shirotsumekusa committed:
Update Tenkuu Danzai Skelter Heaven (primary).ass

Primary edit done

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
 Comment: 0,0:01:31.00,0:01:31.00,Default,chptr,0,0,0,,{OP}OP
 Comment: 0,0:03:19.21,0:03:19.21,Default,chptr,0,0,0,,{Part A}Part A
-Dialogue: 0,0:03:20.24,0:03:23.64,Default,,0,0,0,,Just now, we've received an update.{this is repeated later without たった今 at the start and it's still a complete sentence, hence why "Just now," is at the start}
+Dialogue: 0,0:03:20.24,0:03:23.64,Default,,0,0,0,,This just in: We've received an update.{this is repeated later without たった今 at the start and it's still a complete sentence, hence why "Just now," is at the start}
```
```suggestion
Dialogue: 0,0:03:20.24,0:03:23.64,Default,,0,0,0,,This is just in: We've received an update.{this is repeated later without たった今 at the start and it's still a complete sentence, hence why "Just now," is at the start}
```

Shirotsumekusa replied:
"This just in" is the common phrasing. Resolving without application.

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
-Dialogue: 0,0:04:04.80,0:04:06.31,Default,,0,0,0,,We're on red alert here.
-Dialogue: 0,0:04:06.31,0:04:07.56,Default,,0,0,0,,Haven't you heard the alarms?
-Dialogue: 0,0:04:08.16,0:04:10.06,Default,,0,0,0,,No, I heard them.
+Dialogue: 0,0:04:04.80,0:04:06.31,Default,,0,0,0,,We've sounded red alert.
```
```suggestion
Dialogue: 0,0:04:04.80,0:04:06.31,Default,,0,0,0,,We've sounded the red alert.
```

Shirotsumekusa replied:
"Red alert" is meant more as a status than a type of alert, so the article changes the meaning. Resolving without application.

Shirotsumekusa commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
 Dialogue: 0,0:10:57.60,0:10:59.84,Default,,0,0,0,,Analyzing.
 Dialogue: 0,0:10:59.84,0:11:01.57,Default,,0,0,0,,Core detected in center of object.
-Dialogue: 0,0:11:01.57,0:11:05.12,Default,,0,0,0,,The entity's exterior is heavily armored,\Nso enter from underneath.{上部の表皮は硬いので、脚部まわりから侵入してください}{m - lit. "the outer skin layer/epidermis is hard up top, so enter from the leg area"}
+Dialogue: 0,0:11:01.57,0:11:05.12,Default,,0,0,0,,The entity's shell is heavily armored,\Nso enter from underneath.{上部の表皮は硬いので、脚部まわりから侵入してください}{m - lit. "the outer skin layer/epidermis is hard up top, so enter from the leg area"}
```
shell -> carapace
I knew there was a better word.

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
 Comment: 0,0:04:17.07,0:04:19.54,Default,,0,0,0,,???
-Dialogue: 0,0:04:21.57,0:04:23.07,Default,,0,0,0,,It's a big one, all right.
-Dialogue: 0,0:04:23.07,0:04:24.78,Default,,0,0,0,,Must be around three hundred meters.
+Dialogue: 0,0:04:21.57,0:04:23.07,Default,,0,0,0,,It's one huge sucker, that's for sure.
```
Line length doesn't match the length of the audio. Maybe this?
```suggestion
Dialogue: 0,0:04:21.57,0:04:23.07,Default,,0,0,0,,That's one huge sucker.
```

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
 Comment: 0,0:04:44.00,0:04:50.04,Default,,0,0,0,,... phase three... require... structure
 Dialogue: 0,0:05:06.40,0:05:09.15,Default,,0,0,0,,We've received an emergency summons, Midori.
-Dialogue: 0,0:05:10.70,0:05:11.96,Default,,0,0,0,,I was getting sick of waiting.
+Dialogue: 0,0:05:10.70,0:05:11.96,Default,,0,0,0,,Almost thought they'd never call.
```
```suggestion
Dialogue: 0,0:05:10.70,0:05:11.96,Default,,0,0,0,,I thought they'd never call.
```

Shirotsumekusa committed:
Update Tenkuu Danzai Skelter Heaven (primary).ass

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
-Dialogue: 0,0:05:37.69,0:05:41.29,Default,,0,0,0,,Now look what you've done—\Nit's {\i1}covered{\i0} in mud!{m - not subbing the "mou" before?}{j missed it it timing, but I don't think the line works without a "jeez" or whatever to represent it}
-Dialogue: 0,0:05:41.29,0:05:42.56,Default,,0,0,0,,What's the matter, Rin?
-Dialogue: 0,0:05:43.52,0:05:44.93,Default,,0,0,0,,Oh, right—we're being summoned.
+Dialogue: 0,0:05:37.69,0:05:41.29,Default,,0,0,0,,Look at what you've done—\NI'm all muddy now!{Now look what you've done— it's <\i1>covered<\i0> in mud!}{m - not subbing the "mou" before?}{j missed it it timing, but I don't think the line works without a "jeez" or whatever to represent it}{Na: I think this is Rin talking saying that she's covered in mud.}
```
Agreed here. The voice matches Rin and she got splashed with mud in the earlier shot. The "mou" seems to be covered nicely by the first half of the sentence, so no "Jeez" is necessary.

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
-Dialogue: 0,0:05:43.52,0:05:44.93,Default,,0,0,0,,Oh, right—we're being summoned.
+Dialogue: 0,0:05:37.69,0:05:41.29,Default,,0,0,0,,Look at what you've done—\NI'm all muddy now!{Now look what you've done— it's <\i1>covered<\i0> in mud!}{m - not subbing the "mou" before?}{j missed it it timing, but I don't think the line works without a "jeez" or whatever to represent it}{Na: I think this is Rin talking saying that she's covered in mud.}
+Dialogue: 0,0:05:41.29,0:05:42.56,Default,,0,0,0,,What's up, Rin?
+Dialogue: 0,0:05:43.52,0:05:44.93,Default,,0,0,0,,Oh, right. We're being called-up.
```
There's no hyphen when it's used as a verb.
```suggestion
Dialogue: 0,0:05:43.52,0:05:44.93,Default,,0,0,0,,Oh, right. We're being called up.
```

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
 Dialogue: 0,0:06:17.65,0:06:19.36,Default,,0,0,0,,All agents have arrived.
-Dialogue: 0,0:06:21.36,0:06:24.83,Default,,0,0,0,,Your objective is to annihilate the target.
+Dialogue: 0,0:06:21.36,0:06:24.83,Default,,0,0,0,,Your objective is to eliminate the target.
 Dialogue: 0,0:06:24.83,0:06:27.57,Default,,0,0,0,,Take it out whatever the cost.
```
Either of these would work:
```suggestion
Dialogue: 0,0:06:24.83,0:06:27.57,Default,,0,0,0,,Take it out, whatever the cost.
```
```suggestion
Dialogue: 0,0:06:24.83,0:06:27.57,Default,,0,0,0,,Take it out at whatever cost.
```

Shirotsumekusa replied:
Changing to "Take it out at all costs."

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
+Dialogue: 0,0:06:46.89,0:06:48.89,Default,joined,0,0,0,,Make no mistake. \NFailure is not an option.
+Dialogue: 0,0:06:48.89,0:06:51.42,Default,,0,0,0,,Take it from here, Funagai.
+Dialogue: 0,0:06:52.63,0:06:56.80,Default,,0,0,0,,There are two steps required \Nto neutralize our opponent.
+Dialogue: 0,0:06:56.80,0:06:59.91,Default,,0,0,0,,We must locate its weak spot{discover its weakness} \Nand then destroy the target.
```
The pause in the dialogue lends itself nicely to a comma.
```suggestion
Dialogue: 0,0:06:56.80,0:06:59.91,Default,,0,0,0,,We must locate its weak spot{discover its weakness}, \Nthen destroy the target.
```

Shirotsumekusa replied:
Commas around conjunctions only separate independent clauses. While "Destroy the target" is an independent clause by itself, in this case it's depending on the "We must [then destroy the target]."

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
+Dialogue: 0,0:06:52.63,0:06:56.80,Default,,0,0,0,,There are two steps required \Nto neutralize our opponent.
+Dialogue: 0,0:06:56.80,0:06:59.91,Default,,0,0,0,,We must locate its weak spot{discover its weakness} \Nand then destroy the target.
+Dialogue: 0,0:07:00.93,0:07:05.12,Default,,0,0,0,,We'll get into close range \Nand fire a probe into it with a bazooka,{バズーカで探査弾を至近距離から目標の体内に打ち込み}
+Dialogue: 0,0:07:05.12,0:07:06.80,Default,edit,0,0,0,,verify the location of its logic{master} core,{j - edit: better-sounding term for controling core?}{制御コアを確定}{m - I'm not super sure on how to interpret 確定 here (but I'm sure it *is* 確定, that's what it is in the vn), but I guess it's probably something like finding/determining where the core is (using the probe)?}
```
"Logic" makes this being sound artificial, and I don't think it is.

Shirotsumekusa replied:
https://jisho.org/word/%E5%88%B6%E5%BE%A1
That's the term used for "controlling/master/logic/whatever we go with" here.

The0x539 replied:
Just spitballing: "central nervous system"?

bucket3432 replied:
I had thought about "brain", but I can't picture the light near the end of the episode as being described as one.

bucket3432 replied:
Another option is to drop "controlling" altogether if we think just "core" is enough to convey the meaning. When I see "core" in this context, I think of some sort of central system anyway.

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
-Dialogue: 0,0:07:14.81,0:07:18.75,Default,,0,0,0,,Once its energy readings drop,\Nyou will move out and engage.
-Dialogue: 0,0:07:18.75,0:07:20.32,Default,,0,0,0,,Strike swiftly and retreat.{m - lit. "make sure to act swiftly"}
+Dialogue: 0,0:07:14.81,0:07:18.75,Default,,0,0,0,,Once its energy readings drop, \Nyou will move out and engage the target.
+Dialogue: 0,0:07:18.75,0:07:20.32,Default,,0,0,0,,It is critical we act with haste.
```
Adding "that" improves the rhythm of this line.
```suggestion
Dialogue: 0,0:07:18.75,0:07:20.32,Default,,0,0,0,,It is critical that we act with haste.
```

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
-Dialogue: 0,0:07:18.75,0:07:20.32,Default,,0,0,0,,Strike swiftly and retreat.{m - lit. "make sure to act swiftly"}
+Dialogue: 0,0:07:14.81,0:07:18.75,Default,,0,0,0,,Once its energy readings drop, \Nyou will move out and engage the target.
+Dialogue: 0,0:07:18.75,0:07:20.32,Default,,0,0,0,,It is critical we act with haste.
 Dialogue: 0,0:07:20.32,0:07:23.89,Default,,0,0,0,,Consider this the only chance we've got.{think that if this does not succeed, there will be no next time}
```
Is "we've got" too informal? Nothing else has quite the same impact, though, so maybe we should leave this.

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
 Dialogue: 0,0:07:26.84,0:07:28.22,Default,,0,0,0,,Strike the core.
 Dialogue: 0,0:07:29.48,0:07:30.30,Default,,0,0,0,,Understood.
-Dialogue: 0,0:07:30.30,0:07:31.32,Default,,0,0,0,,Captain!
+Dialogue: 0,0:07:30.30,0:07:31.32,Default,,0,0,0,,Captain—
```
She doesn't finish the word.
```suggestion
Dialogue: 0,0:07:30.30,0:07:31.32,Default,,0,0,0,,Capt—
```

Shirotsumekusa replied:
I hear what seems to be the full "taichou" over the alarm, but honestly, either way it's not conveying her full intent and gets the meaning across.

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
+Dialogue: 0,0:08:00.03,0:08:01.88,Default,,0,0,0,,I can't accept this, Captain.
 Dialogue: 0,0:08:01.88,0:08:03.70,Default,,0,0,0,,Why am {\i1}I{\i0} backup?
-Dialogue: 0,0:08:03.70,0:08:05.46,Default,,0,0,0,,You have a problem with my commands?
+Dialogue: 0,0:08:03.70,0:08:05.46,Default,,0,0,0,,You have a problem with my orders?
```
```suggestion
Dialogue: 0,0:08:03.70,0:08:05.46,Default,,0,0,0,,Do you have a problem with my orders?
```

Shirotsumekusa replied:
I think the colloquial structure helps set the tone here.

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
-Dialogue: 0,0:08:07.97,0:08:09.90,Default,,0,0,0,,Rin, go on ahead.
-Dialogue: 0,0:08:16.90,0:08:19.21,Default,,0,0,0,,Are you fine letting her go?
+Dialogue: 0,0:08:07.97,0:08:09.90,Default,,0,0,0,,Rin, you go on.
+Dialogue: 0,0:08:16.90,0:08:19.21,Default,,0,0,0,,Are you fine sending her out there?
```
The audio suggests we should break this into two sentences. The second might technically be ungrammatical, but you'd say it like this if it were spoken anyway.
```suggestion
Dialogue: 0,0:08:16.90,0:08:19.21,Default,,0,0,0,,Are you fine with that? Sending her out there?
```

Shirotsumekusa replied:
I think it works as is. Splitting it like that reeks of adhering to Japanese structure to me, which we don't really have to do at the cost of making it sound unnatural.

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
-Dialogue: 0,0:08:16.90,0:08:19.21,Default,,0,0,0,,Are you fine letting her go?
+Dialogue: 0,0:08:07.97,0:08:09.90,Default,,0,0,0,,Rin, you go on.
+Dialogue: 0,0:08:16.90,0:08:19.21,Default,,0,0,0,,Are you fine sending her out there?
 Dialogue: 0,0:08:19.21,0:08:20.43,Default,,0,0,0,,What's that supposed to mean?
```
This seems a bit too informal.
```suggestion
Dialogue: 0,0:08:19.21,0:08:20.43,Default,,0,0,0,,What are you saying?
```
It's probably fine, though.

Shirotsumekusa replied:
It is. He's not speaking in particularly formal Japanese.
We'll settle on a brusque "What are you implying?"

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
+Dialogue: 0,0:08:44.77,0:08:46.24,Default,,0,0,0,,{\i1}I'll show him.
+Dialogue: 0,0:08:47.07,0:08:49.22,Default,,0,0,0,,{\i1}I'll {\i0}prove{\i1} I'm superior to Rin.
 Dialogue: 0,0:08:53.61,0:08:58.34,Default,,0,0,0,,You're on point. Ensure you penetrate the target directly with the probe.{？？に装填された探査弾を？？確実に打ち込んで下さい - you can see the scene from the vn here: https://youtu.be/dkFsJWiJVQ8?t=1429  but the audio is almost as inaudible there as here}{j - I can definitely hear ファースト after a relisten}
 Dialogue: 0,0:08:58.34,0:09:00.72,Default,,0,0,0,,With this improvised weapon?
```
This line seems a bit stiff.
```suggestion
Dialogue: 0,0:08:58.34,0:09:00.72,Default,,0,0,0,,With this makeshift weapon?
```

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
 Dialogue: 0,0:09:33.72,0:09:35.99,Default,,0,0,0,,{\i1}No one's going to listen to excuses.
-Dialogue: 0,0:09:35.99,0:09:38.63,Default,,0,0,0,,{\i1}I won't give in to the enemy or my own weakness.
-Dialogue: 0,0:09:38.63,0:09:41.16,Default,,0,0,0,,{\i1}Then I'll tell him how I feel!{literally just "convey to him", but this seems an obvious enough inference}
+Dialogue: 0,0:09:35.99,0:09:38.63,Default,,0,0,0,,{\i1}I won't give in to the enemy \Nor my own weakness.
```
There's a pause in the dialogue that lends itself nicely to an ellipsis (or a comma, but I think an ellipsis works better).
```suggestion
Dialogue: 0,0:09:35.99,0:09:38.63,Default,,0,0,0,,{\i1}I won't give in... not to the enemy, \Nnor my own weakness.
```

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
+Dialogue: 0,0:11:01.57,0:11:05.12,Default,,0,0,0,,The entity's carapace is heavily armored,\Nso enter from underneath.{上部の表皮は硬いので、脚部まわりから侵入してください}{m - lit. "the outer skin layer/epidermis is hard up top, so enter from the leg area"}
 Dialogue: 0,0:11:08.78,0:11:09.73,Default,,0,0,0,,Are you listening, Rin?
-Dialogue: 0,0:11:09.73,0:11:11.35,Default,,0,0,0,,There's no time for emotions.
+Dialogue: 0,0:11:09.73,0:11:11.35,Default,,0,0,0,,Don't let your emotions get the best of you.
```
This doesn't seem harsh enough. I'd revert.
```suggestion
Dialogue: 0,0:11:09.73,0:11:11.35,Default,,0,0,0,,There's no time for emotions.
```

Shirotsumekusa replied:
Seeing the two side-by-side, I agree, but "There's no time for emotions." is pretty awkward.
Settling on "There's no time to get all emotional."

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
-Dialogue: 0,0:11:38.38,0:11:40.69,Default,,0,0,0,,This is an artificial human.
-Dialogue: 0,0:11:40.95,0:11:46.48,Default,,0,0,0,,As it grows, its intelligence will surpass a human's,
-Dialogue: 0,0:11:46.48,0:11:48.38,Default,,0,0,0,,along with its physical abilities.{m - lit. "and the same goes for its physical abilities"}
+Dialogue: 0,0:11:38.38,0:11:40.69,Default,,0,0,0,,This is a homunculus.
```
"Homunculus" has a different meaning in the scientific and technical space than in the pseudoscience space. I vote to revert.
```suggestion
Dialogue: 0,0:11:38.38,0:11:40.69,Default,,0,0,0,,This is an artificial human.
```

The0x539 replied:
"Synthetic", perhaps? Might convey a bit more meaning.

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
 Dialogue: 0,0:12:22.17,0:12:23.38,Default,,0,0,0,,Not at all.
-Dialogue: 0,0:12:23.38,0:12:27.00,Default,,0,0,0,,It's just... I'm an artificial being.
-Dialogue: 0,0:12:27.00,0:12:29.23,Default,,0,0,0,,My lifespan is different to yours.
+Dialogue: 0,0:12:23.38,0:12:27.00,Default,,0,0,0,,It's just... I'm a homunculus.
```
In the same vein as before:
```suggestion
Dialogue: 0,0:12:23.38,0:12:27.00,Default,,0,0,0,,It's just... I'm artificial.
```
Or:
```suggestion
Dialogue: 0,0:12:23.38,0:12:27.00,Default,,0,0,0,,It's just... I wasn't made naturally.
```
Or revert:
```suggestion
Dialogue: 0,0:12:23.38,0:12:27.00,Default,,0,0,0,,It's just... I'm an artificial being.
```

Shirotsumekusa replied:
Changed to "It's just... I'm not a real human."

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
-Dialogue: 0,0:12:39.01,0:12:40.49,Default,,0,0,0,,There's nothing to be worried about.
-Dialogue: 0,0:12:41.77,0:12:45.46,Default,,0,0,0,,However, we won't know your lifespan without more research.
-Dialogue: 0,0:12:46.75,0:12:50.50,Default,,0,0,0,,As the first generation,\Nyou are the research subjects.
+Dialogue: 0,0:12:35.24,0:12:39.01,Default,,0,0,0,,The organization is responsible \Nfor all of your well-beings.
```
"Well-being" is uncountable.
```suggestion
Dialogue: 0,0:12:35.24,0:12:39.01,Default,,0,0,0,,The organization is responsible \Nfor your well-being.
```
```suggestion
Dialogue: 0,0:12:35.24,0:12:39.01,Default,,0,0,0,,The organization is responsible \Nfor your collective well-being.
```

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
+Dialogue: 0,0:12:35.24,0:12:39.01,Default,,0,0,0,,The organization is responsible \Nfor all of your well-beings.
+Dialogue: 0,0:12:39.01,0:12:40.49,Default,,0,0,0,,Don't stress about it.
+Dialogue: 0,0:12:41.77,0:12:45.46,Default,,0,0,0,,However, we won't know your lifespans \Nwithout more research.
+Dialogue: 0,0:12:46.75,0:12:50.50,Default,,0,0,0,,As the first generation of homunculi, \Nyou girls are the prototypes.
```
```suggestion
Dialogue: 0,0:12:46.75,0:12:50.50,Default,,0,0,0,,As the first generation of artificial humans, \Nyou girls are the prototypes.
```

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
 Dialogue: 0,0:12:55.49,0:12:56.79,Default,,0,0,0,,All right?
 Dialogue: 0,0:12:56.79,0:12:57.98,Default,,0,0,0,,Understood.
 Dialogue: 0,0:12:59.78,0:13:00.99,Default,,0,0,0,,That's good.
 Dialogue: 0,0:13:00.99,0:13:03.35,Default,,0,0,0,,Your orders are absolute.
```
"Absolute" seems a bit stiff, but she's being stiff, too. I'd go with "final" if we had to change it out, but I think we're okay.

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
 Dialogue: 0,0:12:59.78,0:13:00.99,Default,,0,0,0,,That's good.
 Dialogue: 0,0:13:00.99,0:13:03.35,Default,,0,0,0,,Your orders are absolute.
-Dialogue: 0,0:13:03.35,0:13:06.46,Default,edit,0,0,0,,You're as dutiful as they come, aren't you?{like the impulse to question his orders would never enter her mind}
+Dialogue: 0,0:13:03.35,0:13:06.46,Default,edit,0,0,0,,You don't have an insubordinate \Nbone in your body, do you?{You're as dutiful as they come, aren't you?}{like the impulse to question his orders would never enter her mind}
```
I see what you're trying to do there, but I'm not sure the phrase works. I can't come up with anything better, though.

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
+Dialogue: 0,0:13:35.17,0:13:38.77,Default,,0,0,0,,We know its weak spot now, \Nso all I've gotta do is ram into it with my unit!{弱点さえ分かれば、こいつで突っ込むのもわけないさ}{pretty sure he means as a suicide attack, with the alternative in his mind being one of the girls doing the same thing}
+Dialogue: 0,0:13:38.77,0:13:40.73,Default,,0,0,0,,Funagai! Have you lost your mind?
+Dialogue: 0,0:13:42.17,0:13:43.90,Default,,0,0,0,,I told myself I was just doing my job.
+Dialogue: 0,0:13:45.61,0:13:48.71,Default,,0,0,0,,I coped with everything by convincing \Nmyself that was true.
```
Seems a bit awkward to me to break it up there. I'd probably break before "convincing" or before "that", but the latter would be too imbalanced. I think you need an extra "that" after "myself", but then you end up with a double "that". Perhaps it needs rewriting:
```suggestion
Dialogue: 0,0:13:45.61,0:13:48.71,Default,,0,0,0,,I coped with everything by \Nconvincing myself of that.
```

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
+Dialogue: 0,0:13:42.17,0:13:43.90,Default,,0,0,0,,I told myself I was just doing my job.
+Dialogue: 0,0:13:45.61,0:13:48.71,Default,,0,0,0,,I coped with everything by convincing \Nmyself that was true.
+Dialogue: 0,0:13:49.37,0:13:52.71,Default,,0,0,0,,But thinking back on it...
+Dialogue: 0,0:13:53.21,0:13:54.40,Default,,0,0,0,,all I can see is their smiles.
```
Need to flag this one for extra lead-out time now that it's longer
```suggestion
Dialogue: 0,0:13:53.21,0:13:54.40,Default,TM,0,0,0,,all I can see is their smiles.
```

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
+Dialogue: 0,0:13:38.77,0:13:40.73,Default,,0,0,0,,Funagai! Have you lost your mind?
+Dialogue: 0,0:13:42.17,0:13:43.90,Default,,0,0,0,,I told myself I was just doing my job.
+Dialogue: 0,0:13:45.61,0:13:48.71,Default,,0,0,0,,I coped with everything by convincing \Nmyself that was true.
+Dialogue: 0,0:13:49.37,0:13:52.71,Default,,0,0,0,,But thinking back on it...
```
The pause lends itself to an ellipsis.
```suggestion
Dialogue: 0,0:13:49.37,0:13:52.71,Default,,0,0,0,,But... thinking back on it...
```

Shirotsumekusa replied:
The prevalence of ellipses in fansubbing is frankly disturbing. It functions fine without it.

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
+Dialogue: 0,0:13:45.61,0:13:48.71,Default,,0,0,0,,I coped with everything by convincing \Nmyself that was true.
+Dialogue: 0,0:13:49.37,0:13:52.71,Default,,0,0,0,,But thinking back on it...
+Dialogue: 0,0:13:53.21,0:13:54.40,Default,,0,0,0,,all I can see is their smiles.
+Dialogue: 0,0:13:56.11,0:13:58.30,Default,,0,0,0,,So, I... I—{Na: For some reason I get the feeling he's trying to say that he loves these robochicks, so making this a little more vague.}
```
He gets cut off but not really interrupted. I'd opt for an ellipsis here.
```suggestion
Dialogue: 0,0:13:56.11,0:13:58.30,Default,,0,0,0,,So, I... I...{Na: For some reason I get the feeling he's trying to say that he loves these robochicks, so making this a little more vague.}
```

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
 Dialogue: 0,0:14:13.50,0:14:15.93,Default,,0,0,0,,Rin! What are you doing?
-Dialogue: 0,0:14:15.93,0:14:20.34,Default,,0,0,0,,I didn't follow your orders all this time because of how I was created!
-Dialogue: 0,0:14:20.72,0:14:23.94,Default,,0,0,0,,It was because I loved you!
+Dialogue: 0,0:14:15.93,0:14:20.34,Default,,0,0,0,,I haven't followed your orders just \Nbecause I was created to do so!
```
Awkward line break. I would also use the past continuous here.
```suggestion
Dialogue: 0,0:14:15.93,0:14:20.34,Default,,0,0,0,,I haven't been following your orders \Njust because I was created to do so!
```
Alternatively, to stick closer to the original TL:
```suggestion
Dialogue: 0,0:14:15.93,0:14:20.34,Default,,0,0,0,,I wasn't following your orders \Njust because I was created to do so!
```

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
-Dialogue: 0,0:14:15.93,0:14:20.34,Default,,0,0,0,,I didn't follow your orders all this time because of how I was created!
-Dialogue: 0,0:14:20.72,0:14:23.94,Default,,0,0,0,,It was because I loved you!
+Dialogue: 0,0:14:15.93,0:14:20.34,Default,,0,0,0,,I haven't followed your orders just \Nbecause I was created to do so!
+Dialogue: 0,0:14:20.72,0:14:23.94,Default,,0,0,0,,I've done it because I love you!
```
If you change it, keep the tense.
```suggestion
Dialogue: 0,0:14:20.72,0:14:23.94,Default,,0,0,0,,I've been following them because I love you!
```
If you took the alternative suggestion:
```suggestion
Dialogue: 0,0:14:20.72,0:14:23.94,Default,,0,0,0,,I was following them because I love you!
```

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
-Dialogue: 0,0:15:31.44,0:15:35.38,Default,,0,0,0,,Certainly, some humans are slaves to their emotions.
-Dialogue: 0,0:15:35.38,0:15:39.04,Default,,0,0,0,,And their history is one of destruction, no end in sight.
+Dialogue: 0,0:15:06.22,0:15:09.96,Default,,0,0,0,,Skelter Titan and Skelter Heaven \Nhave penetrated the target.
+Dialogue: 0,0:15:11.72,0:15:14.34,Default,joined,0,0,0,,{\i1}What on earth... {Line 2: What is this?}
```
Joining is fine, but we need to preserve the pause somehow.

Shirotsumekusa replied:
The audio does a fine job of preserving the pause. The subtitles don't have to reflect that. If these were closed captions, that'd be one thing, but they're not.

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
-Dialogue: 0,0:15:35.38,0:15:39.04,Default,,0,0,0,,And their history is one of destruction, no end in sight.
+Dialogue: 0,0:15:06.22,0:15:09.96,Default,,0,0,0,,Skelter Titan and Skelter Heaven \Nhave penetrated the target.
+Dialogue: 0,0:15:11.72,0:15:14.34,Default,joined,0,0,0,,{\i1}What on earth... {Line 2: What is this?}
+Dialogue: 0,0:15:21.98,0:15:24.14,Default,,0,0,0,,This is its logic{master} core.
```
Flagging this for "logic core".

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
+Dialogue: 0,0:15:11.72,0:15:14.34,Default,joined,0,0,0,,{\i1}What on earth... {Line 2: What is this?}
+Dialogue: 0,0:15:21.98,0:15:24.14,Default,,0,0,0,,This is its logic{master} core.
+Dialogue: 0,0:15:24.95,0:15:28.03,Default,,0,0,0,,It's speaking directly into my mind.
+Dialogue: 0,0:15:31.44,0:15:35.38,Default,,0,0,0,,It's true some humans are slaves \Nto their emotions.
```
This needs a comma, both because of a pause and for grammar.
```suggestion
Dialogue: 0,0:15:31.44,0:15:35.38,Default,,0,0,0,,It's true, some humans are slaves \Nto their emotions.
```

Shirotsumekusa replied:
Commas for pauses are ungrammatical. If we were to punctuate here, a colon or a period separating the two clauses would be the correct mark. It works fine together as one independent clause, but it could also function as two so I'm not against the colon if you feel it'd read more the way you hear it in your head.

bucket3432 replied:
Perhaps it's because I have a tendency to insert the optional "that" more often than not. You're right about the punctuation (and I'm going to blame that on how late it was when I wrote this). I'd be good with either a colon or inserting a "that".
```suggestion
Dialogue: 0,0:15:31.44,0:15:35.38,Default,,0,0,0,,It's true that some humans are slaves \Nto their emotions.
```

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
-Dialogue: 0,0:15:50.78,0:15:52.96,Default,,0,0,0,,But it's not for humanity.
-Dialogue: 0,0:15:52.96,0:15:55.62,Default,,0,0,0,,It's for the one person I want to protect!
-Dialogue: 0,0:15:55.62,0:15:57.44,Default,,0,0,0,,That's why I'm fighting!
+Dialogue: 0,0:15:50.78,0:15:52.96,Default,,0,0,0,,Not for the sake of mankind as a whole,
```
There's an ううん right before this that didn't get translated/timed and I can't unhear it.
```suggestion
Dialogue: 0,0:15:49.53,0:15:50.78,Default,,0,0,0,,No...
Dialogue: 0,0:15:50.78,0:15:52.96,Default,,0,0,0,,Not for the sake of mankind as a whole,
```

Shirotsumekusa replied:
This change makes it sound like she's talking to herself instead of responding to the being. Modified in script to be one whole line.

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
+Dialogue: 0,0:15:50.78,0:15:52.96,Default,,0,0,0,,Not for the sake of mankind as a whole,
+Dialogue: 0,0:15:52.96,0:15:55.62,Default,,0,0,0,,but for the one person I care about most!
+Dialogue: 0,0:15:55.62,0:15:57.44,Default,,0,0,0,,I fight for that one man!
 Dialogue: 0,0:16:31.44,0:16:33.53,Default,,0,0,0,,Yes, that's exactly right.
```
This feels more natural to me:
```suggestion
Dialogue: 0,0:16:31.44,0:16:33.53,Default,,0,0,0,,Yes, it is as you say.
```

Shirotsumekusa replied:
He's not responding to her, if that's what these changes are implying. From what I could tell, he's debriefing the situation to presumably someone higher up the food chain.

bucket3432 replied:
The English of both changes can imply that he's responding to something that the person on the other side of the phone has said, and it must be a response to something because of the presence of "yes". That said, I can see how the first can be more of an assertion responding to clarification ("This thing happened." -> "Did it really?" -> "Yes, it really did."), whereas the second may imply more that he's receiving information.

"That's exactly right" still seems awkward to me, so perhaps this is a better wording that preserves the nuances of the first?
```suggestion
Dialogue: 0,0:16:31.44,0:16:33.53,Default,,0,0,0,,Yes, that is the case.
```

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
+Dialogue: 0,0:16:40.94,0:16:43.60,Default,,0,0,0,,The problem lies in how we humans used them.
+Dialogue: 0,0:16:44.39,0:16:47.02,Default,,0,0,0,,It always does.
 Dialogue: 0,0:17:00.80,0:17:02.17,Default,,0,0,0,,Breaking news.
 Dialogue: 0,0:17:02.17,0:17:06.55,Default,,0,0,0,,Multiple mysterious entities have appeared in Tokyo's airspace.
```
```suggestion
Dialogue: 0,0:17:02.17,0:17:06.55,Default,,0,0,0,,Multiple mysterious entities \Nhave appeared in Tokyo's airspace.
```

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
+Dialogue: 0,0:03:33.16,0:03:38.16,Signs,,0,0,0,,{\pos(64,178.286)}Mysterious Airborne Entity Begins Moving Over Tokyo Airspace
+Dialogue: 0,0:03:33.16,0:03:38.16,Signs,,0,0,0,,{\pos(449.143,183.429)}Mysterious Airborne Entity Begins Moving Over Tokyo Airspace
+Dialogue: 0,0:16:54.86,0:16:59.76,Signs,,0,0,0,,{\pos(325.143,254.286)}One Year Later
+Dialogue: 0,0:17:02.10,0:17:11.98,Signs,,0,0,0,,{\pos(283.429,408.571)}Mysterious Airborne Entity Appears over Tokyo Again
```
I don't think you had meant to revert these lines.
```suggestion
Comment: 0,0:03:23.72,0:03:33.19,Signs,,0,0,0,,# News headline
Dialogue: 0,0:03:23.72,0:03:33.19,Signs,,0,0,0,,{\pos(311.429,397.714)}Mysterious airborne entity begins moving over Tokyo airspace
Comment: 0,0:03:33.19,0:03:38.20,Signs,,0,0,0,,# News headline top left
Dialogue: 0,0:03:33.19,0:03:38.20,Signs,,0,0,0,,{\pos(64,178.286)}Mysterious airborne entity begins moving over Tokyo airspace
Comment: 0,0:03:33.19,0:03:38.20,Signs,,0,0,0,,# News headline top right
Dialogue: 0,0:03:33.19,0:03:38.20,Signs,,0,0,0,,{\pos(449.143,183.429)}Mysterious airborne entity begins moving over Tokyo airspace
Comment: 0,0:16:54.86,0:16:59.76,Signs,,0,0,0,,# One year later
Dialogue: 0,0:16:54.86,0:16:59.76,Signs,,0,0,0,,{\pos(325.143,254.286)}One year later...
Comment: 0,0:17:02.37,0:17:11.98,Signs,,0,0,0,,# News headline reprise
Dialogue: 0,0:17:02.37,0:17:11.98,Signs,,0,0,0,,{\pos(283.429,408.571)}Mysterious airborne entity appears over Tokyo once more
```

Shirotsumekusa replied:
I used a dated file for the edit, so reverting the signs was definitely unintentional. I'll check it out for consistency and flow before the night ends.

Shirotsumekusa committed:
Update primary/Tenkuu Danzai Skelter Heaven (primary).ass

Co-Authored-By: bucket3432 <bucket3432@disroot.org>

Shirotsumekusa committed:
Update Tenkuu Danzai Skelter Heaven (primary).ass

Comments cleaned up and minor changes here and there.

Shirotsumekusa committed:
Merge branch 'pri/edit' of https://github.com/butterfansubs/skelter-heaven into pri/edit

Shirotsumekusa committed:
Update primary/Tenkuu Danzai Skelter Heaven (primary).ass

Co-Authored-By: bucket3432 <bucket3432@disroot.org>

Shirotsumekusa committed:
Update primary/Tenkuu Danzai Skelter Heaven (primary).ass

Co-Authored-By: bucket3432 <bucket3432@disroot.org>

Shirotsumekusa committed:
Update Tenkuu Danzai Skelter Heaven (primary).ass

Shirotsumekusa committed:
Update primary/Tenkuu Danzai Skelter Heaven (primary).ass

Co-Authored-By: bucket3432 <bucket3432@disroot.org>

Shirotsumekusa committed:
Update Tenkuu Danzai Skelter Heaven (primary).ass

Shirotsumekusa committed:
Merge branch 'pri/edit' of https://github.com/butterfansubs/skelter-heaven into pri/edit

Shirotsumekusa committed:
Update primary/Tenkuu Danzai Skelter Heaven (primary).ass

Co-Authored-By: bucket3432 <bucket3432@disroot.org>

Shirotsumekusa committed:
Update primary/Tenkuu Danzai Skelter Heaven (primary).ass

Ah, golden. I hated this too but my brain wouldn't pull away from "ramshackled" and "ragtag," neither of which felt right.

Co-Authored-By: bucket3432 <bucket3432@disroot.org>

Shirotsumekusa committed:
Update primary/Tenkuu Danzai Skelter Heaven (primary).ass

I was on the fence about this too. It reads nicer but takes on a more literary meaning with homunculus.

Co-Authored-By: bucket3432 <bucket3432@disroot.org>

Shirotsumekusa committed:
Update primary/Tenkuu Danzai Skelter Heaven (primary).ass

Co-Authored-By: bucket3432 <bucket3432@disroot.org>

Shirotsumekusa committed:
Update primary/Tenkuu Danzai Skelter Heaven (primary).ass

Co-Authored-By: bucket3432 <bucket3432@disroot.org>

Shirotsumekusa committed:
Update Tenkuu Danzai Skelter Heaven (primary).ass

Applying edit suggestions.

Shirotsumekusa committed:
Merge branch 'pri/edit' of https://github.com/butterfansubs/skelter-heaven into pri/edit

Shirotsumekusa committed:
Update primary/Tenkuu Danzai Skelter Heaven (primary).ass

Co-Authored-By: bucket3432 <bucket3432@disroot.org>

Shirotsumekusa committed:
Update Tenkuu Danzai Skelter Heaven (primary).ass

Shirotsumekusa committed:
Merge branch 'pri/edit' of https://github.com/butterfansubs/skelter-heaven into pri/edit

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,100 +159,89 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
+Dialogue: 0,0:01:47.85,0:01:49.36,Signs,,0,0,0,,{\pos(322.545,320.302)}AA passed many feature and years\Nand succeededin producing the first\Ngeneration at least
+Dialogue: 0,0:01:48.45,0:01:49.25,Signs,,0,0,0,,{\pos(498.4,420)}To the sake when coming
+Dialogue: 0,0:01:53.03,0:02:07.57,Signs,,0,0,0,,{\pos(69.212,197.635)}Even if they were called "Weattprhopet"{sic, from ウェットプロープ} and gave the complete same\Nprograms, become altogetherdifferent capability balance
+Dialogue: 0,0:02:07.87,0:02:29.13,Signs,,0,0,0,,{\pos(572,219.333)}She "Weattprhopet" got into "Skelter" united with each individuality, and has\Nrepeted{"repeated"} the improvement constantly in preparation for the day which should come.
```
This still appears to be reverted to the pre-edit state, so make sure this doesn't make it as-is in when we merge it.

bucket3432 reviewed (commented):
The only thing I haven't replied to yet is the stuff about the "logic core". I'm going to see what Null Peta used and get back to it.

Shirotsumekusa committed:
Update primary/Tenkuu Danzai Skelter Heaven (primary).ass

Co-Authored-By: bucket3432 <bucket3432@disroot.org>

Shirotsumekusa committed:
Update Tenkuu Danzai Skelter Heaven (primary).ass

Shirotsumekusa committed:
Update primary/Tenkuu Danzai Skelter Heaven (primary).ass

Co-Authored-By: bucket3432 <bucket3432@disroot.org>

Shirotsumekusa committed:
Update Tenkuu Danzai Skelter Heaven (primary).ass

Neurocore and other minor adjustments.

Shirotsumekusa committed:
Update Tenkuu Danzai Skelter Heaven (primary).ass

Added/removed few markers and changes one line for flow.

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
+Dialogue: 0,0:06:52.63,0:06:56.80,Default,,0,0,0,,There are two steps required \Nto neutralize our opponent.
+Dialogue: 0,0:06:56.80,0:06:59.91,Default,,0,0,0,,We must locate its weak spot{discover its weakness} \Nand then destroy the target.
+Dialogue: 0,0:07:00.93,0:07:05.12,Default,,0,0,0,,We'll get into close range \Nand fire a probe into it with a bazooka,
+Dialogue: 0,0:07:05.12,0:07:06.80,Default,,0,0,0,,verify the location of its neurocore core,
```
Redundant phrasing.
```suggestion
Dialogue: 0,0:07:05.12,0:07:06.80,Default,,0,0,0,,verify the location of its neurocore,
```

bucket3432 commented on primary/Tenkuu Danzai Skelter Heaven (primary).ass:
```diff
@@ -161,79 +159,77 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
-Dialogue: 0,0:13:00.99,0:13:03.35,Default,,0,0,0,,Your orders are absolute.
-Dialogue: 0,0:13:03.35,0:13:06.46,Default,edit,0,0,0,,You're as dutiful as they come, aren't you?{like the impulse to question his orders would never enter her mind}
+Dialogue: 0,0:13:00.99,0:13:03.35,Default,,0,0,0,,Your orders are final.
+Dialogue: 0,0:13:03.35,0:13:06.46,Default,,0,0,0,,You don't have an insubordinate \Nbone in your body, do you?
```
I finally thought of a phrase that might be clearer:
```suggestion
Dialogue: 0,0:13:03.35,0:13:06.46,Default,,0,0,0,,You don't have an inkling of insubordination, do you?
```
Optionally with "in you" after "insubordination":
```suggestion
Dialogue: 0,0:13:03.35,0:13:06.46,Default,,0,0,0,,You don't have an inkling of insubordination in you, do you?
```

Shirotsumekusa committed:
Update primary/Tenkuu Danzai Skelter Heaven (primary).ass

Co-Authored-By: bucket3432 <bucket3432@disroot.org>
